src/sidebar/components/focused-mode-header.js

```diff
@@ -8,27 +8,21 @@ import useStore from '../store/use-store';
  * and abstracted if needed. Allow user to toggle in and out of the focus "mode."
  */
 export default function FocusedModeHeader() {
-  const actions = useStore(store => ({
-    setFocusModeFocused: store.setFocusModeFocused,
-  }));
+  const toggleFocusMode = useStore(store => store.toggleFocusMode);
   const selectors = useStore(store => ({
-    focusModeFocused: store.focusModeFocused(),
-    focusModeHasUser: store.focusModeHasUser(),
+    focusModeActive: store.focusModeActive(),
```

Comment:
I've changed what was previously "enabled" and "focused" to "configured" and "active" respectively throughout these changes. I find `focusModeActive()` and `focusModeConfigured()` easier to grok quickly than `focusModeFocused()` and `focusModeEnabled()`, respectively, e.g.